### PR TITLE
DOC: Note on token generation from command line

### PR DESCRIPTION
**Pull request by teddyrendahl**
Some breadcrumbs for generating a token via the command line. Certainly possible to do this in `build.py` but probably not worth the time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,22 @@ another.
 - Make a PR and explain what the package is for
 
 ## Building
-- Generate a valid anaconda token, preferrably one that will expire soon
-- Activate a conda environment with `conda-build=3.5.1` and `anaconda-client`
-  installed
-- `./build.py --channel pcds-tag --token $ANACONDA_TOKEN`
+`build.py` will slowly check all the package variants in the build matrix and
+build the variants that are missing on `pcds-tag`. Yes, we can make this faster
+with some effort.
```
Comment (ZLLentz) [inline README.md:26]: I actually did make this faster! But I guess it's technically still slow.

File: README.md [hunk 2]
```diff
@@ -21,11 +21,22 @@ another.
...
+### Requirements
+- You must have the `anaconda-client` package installed as well as be in an
+  Conda environment with `conda-build=3.5.1`
+- You must have a valid `Anaconda.org` profile that has access to the
+  `pcds-tag` organization
+
+### How To
+- Generate a valid Anaconda token
+```bash
+$ANACONDA_TOKEN=$(anaconda auth --create --name build --org pcds-tag --scopes api --max-age 1800)
```
Comment (ZLLentz) [inline README.md:37]: Is max-age in seconds? This looks good to me.

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
TBH we'd probably include this in `build.py` using the `subprocess` module. Not a big deal to include right away. Thanks for the update, merging docs changes.

File: README.md (hunk 1 above)
Comment (teddyrendahl) [reply README.md:26]: Want me to take it out? Slow is always relative ...

File: README.md (hunk 2 above)
Comment (teddyrendahl) [reply README.md:37]: Yup. 1/2 hour seemed about the right amount of time, but I guess there is no harm in making it longer.

File: README.md (hunk 1 above)
Comment (ZLLentz) [reply README.md:26]: Nah, best to keep expectations low

**Pull request merged**